prac_06/programming_language.py

```diff
@@ -0,0 +1,19 @@
+"""CP1404 practical Programming language class"""
+
+
+class ProgrammingLanguage:
+
+    def __init__(self, field, typing, reflection, year):
+        self.field = field
```

Comment:
You've misread the table. "Field" is a heading for all the fields, not the "name" of the language.